routes/api/books.js

```diff
@@ -0,0 +1,7 @@
+import express from 'express';
+import { handleNewBook } from '../../controllers/booksController.js';
+const router = express.Router();
+
+router.post('/', handleNewBook);
```

Comment:
From what I understand this from the express package, it adds a POST route that uses the `handleNewBook` function- we name the URL in index.js (I'm pretty sure)